### ISSUE TITLE
Add option to hide KeePassXC to tray after entry level Auto-Type 

### ISSUE THREAD

**Issue by zavjah** [keepassxreboot/keepassxc#8766]
Hi,

until few days ago I was using PasswordSafe and very much liked the option, that the DB was closed, not only locked, automatically upon auto text. Also, I could specify that after time out of X sec the DB is not only locked but closed, too.

Now, I am switching to KeePassXC due to many more functions, more frequent updates and also much better UI.  It would be awesome to have this options in KeePassXC, too.

Cheers and thx for great works,
zavjah

**Comment by droidmonkey** [comment]
It doesn't make any sense to close the database, what use case do you have where you want to constantly reopen your database after using it?

I am leaning on a strong no to this option.

**Comment by zavjah** [comment]
Hello droidmonkey, I get your point. Now that you mention it, I think I wrongly described what I'd like to have. Let me rephrase it. 
An option to lock the DB after AutoText is already there so what I'd like to have is an option to "_Minimize to Tray after AutoText_", so the application is not constantly in the task bar.

**Comment by droidmonkey** [comment]
If keepassxc is already in the tray and you use global Auto-Type then it will stay there. I suppose you mean when you use entry level Auto-Type or Ctrl+Shift+V

**Comment by zavjah** [comment]
As far as I can tell in my v2.7.5 this works now perfectly. After the auto-type on an entry level the DB is hidden. I'll close this.
